### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -7231,22 +7231,27 @@ ilustrate->illustrate
 ilustrated->illustrated
 ilustration->illustration
 imagenary->imaginary
+imager->image
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: https://en.wiktionary.org/wiki/imager

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -9413,8 +9415,10 @@ imediate->immediate
 imediately->immediately
 imediatly->immediately
 imense->immense
+imfamus->infamous
 imidiately->immediately
 imigrant->immigrant, emigrant,
+imigrate->immigrate
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Suggest emigrate

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

**Pull request merged**